src/Voron/Platform/Pal.cs

```diff
@@ -46,7 +46,7 @@ static Pal()
 
             try
             {
-                if (File.Exists(toFilename) == false)
+                if (File.Exists(fromFilename))
                     File.Move(fromFilename, toFilename);
```

Comment:
Why did you made this change?

Reply:
because of https://issues.hibernatingrhinos.com/issue/RavenDB-12660#focus=streamItem-67-33793-0-0

If someone copied new lib, we will move it to the general name. This way we avoid unnecessary copy tries and do it once per publish

Reply:
I see, okay